test/js/node/string-module.test.js

```diff
@@ -0,0 +1,15 @@
+import { expect, test } from "bun:test";
+
+test("should import and execute ES module from string", async () => {
```

Comment:
Can you add tests for when parsing the module fails that it throws a `BuildMessage` and has a source url line?